sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -120,39 +117,6 @@ export type MessageIterator<ContextT> = AsyncIterable<MessageAndContext<ContextT
 
 export { MessagingError }
 
-// @public
-export interface NonSessionReceiver<LockModeT extends "peekLock" | "receiveAndDelete"> {
-    // (undocumented)
-    close(): Promise<void>;
-    close(): Promise<void>;
-    diagnostics: {
-        peek(maxMessageCount?: number): Promise<ReceivedMessage[]>;
-        peekBySequenceNumber(fromSequenceNumber: Long, maxMessageCount?: number): Promise<ReceivedMessage[]>;
-    };
-    // (undocumented)
-    entityPath: string;
-    // (undocumented)
-    entityType: "queue" | "subscription";
-    // (undocumented)
```

Comment:
Can you copy the docs from the deleted `ServidceBusReceiverClient`(still in master), and generate the report again!